cmd/podman/ps.go

```diff
@@ -344,7 +344,9 @@ func psCmd(c *cli.Context) error {
 		// Output Namespace headers
 		fmt.Fprintf(w, "\n%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s", hid, hnames, nspid, nscgroup, nsipc, nsmnt, nsnet, nspidns, nsuserns, nsuts)
 	}
-
+	if len(pss) == 0 {
+		fmt.Fprint(w, "\n")
+	}
```

Comment:
I'm assuming this was an intended add on.

Reply:
yeah, @mheon noted that there is no newline after the ps header.

Reply:
cool, thanks.